sklearn/preprocessing/_data.py

```diff
@@ -1538,7 +1538,7 @@ def transform(self, X):
 
         Returns
         -------
-        XP : np.ndarray or CSR/CSC sparse matrix, shape [n_samples, NP]
+        XP : {np.ndarray, CSR/CSC sparse matrix}, shape (n_samples, NP)
```

Comment:
I think that we should put CSR and CSC information in the description rather than in the type.